moki-daniel/Questioner-UI
# Ft-delete-meetupname-promptbox-164044422

**What does this PR do?**
This pull request creates a prompt and confirm pop-ups after the user clicks on the delete a meetup button.

**Description of tasks to be completed?**
The following tasks are completed
-Change the admin-dashboard sidebar links to buttons,
-Add a javascript file for the prompt and confirm box,
-style the buttons and link delete button to the pop-up boxes,

**How should this be manually tested?**
Clone this Repo into your computer, run the admindashboard.html file in the repo using any browser, then you will be able to view the delete button on the left sidebar of the page, click the button and the enter meetup name prompt box appears, after keying in the meetup name and clicking Ok button then confirm popup appears which when clicked the Admin deletes the meetup.

**Pivotal tracker stories associated with the PR?**
#164044422

**Any background images associated with the PR?**
![prompt](https://user-images.githubusercontent.com/5906691/53201562-66f4da00-3635-11e9-965d-6a42671be292.png)
![prompt1](https://user-images.githubusercontent.com/5906691/53201621-8ab82000-3635-11e9-8122-16efadaaac8f.png)
